packages/terra-tag/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-tag",
+  "main": "lib/Tag.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
```

Comment:
Missing a package description

Reply:
Added the package description [here](https://github.com/cerner/terra-core/pull/1306/commits/0c5264f62462ae9bc4cbedbae83c5cac42c28580) .